clarity/src/vm/tests/simple_apply_eval.rs

```diff
@@ -1276,6 +1276,41 @@ fn test_stx_ops_errors() {
     }
 }
 
+#[test]
+fn test_bitwise() {
+    let tests = [
+        "(& 24 16)",
+        "(& u24 u16)",
+        "(^ 24 4)",
+        "(^ u24 u4)",
+        "(| 128 16)",
+        "(| u128 u16)",
+        "(~ 128)",
+        "(~ u128)",
+        "(~ u340282366920938463463374607431768211327)",
+        "(>> u128 u2)",
+        "(<< u4 u2)"
```

Comment:
Can you add test coverage for left- and right-shifting signed integers, including negative ones?

Reply:
Actually, can you add tests for negative integers for all bitwise operators?

Reply:
👍